stage-ovos/00-build/files/etc/mycroft/mycroft.conf

```diff
@@ -24,21 +24,47 @@
          "ovos-vad-plugin-webrtcvad": {"vad_mode": 3}
          },
       "retry_mic_init" : false,
-      "wake_word": "hey_mycroft"
+      "wake_word": "hey_mycroft",
+      "wake_word_upload": {
+         "disable": true}
       },
    "hotwords": {
       "hey_mycroft": {
          "module": "ovos-ww-plugin-precise-lite",
          "model": "https://github.com/OpenVoiceOS/precise-lite-models/raw/master/wakewords/en/hey_mycroft.tflite",
+         "expected_duration": 3,
+         "trigger_level": 3,
+         "sensitivity": 0.5,
          "listen": true,
-         "sound": "snd/start_listening.wav",
+         "fallback_ww": "hey_mycroft_precise"
+         },
+      "hey_mycroft_precise": {
+         "module": "ovos-ww-plugin-precise",
+         "version": "0.3",
+         "model": "https://github.com/MycroftAI/precise-data/raw/models-dev/hey-mycroft.tar.gz",
          "expected_duration": 3,
          "trigger_level": 3,
-         "sensitivity": 0.5
-         }
+         "sensitivity": 0.5,
+         "listen": true,
+         "fallback_ww": "hey_mycroft_vosk"
+         },
+      "hey_mycroft_vosk": {
+         "module": "ovos-ww-plugin-vosk",
+         "samples": ["hey mycroft", "hey microsoft", "hey mike roft", "hey minecraft"],
+         "rule": "fuzzy",
+         "listen": true,
+         "fallback_ww": "hey_mycroft_pocketsphinx"
+         },
+      "hey_mycroft_pocketsphinx": {
+         "module": "ovos-ww-plugin-pocketsphinx",
+         "phonemes": "HH EY . M AY K R AO F T",
+         "threshold": 1e-90,
+         "lang": "en-us",
+         "listen": true
+         },
       },
    "stt": {
-      "fallback_module": ""
+      "fallback_module": "dummy"
```

Comment:
note: this requires the latest alpha version of OPM, so might be safer to leave an empty string unless you explicitly require the min version of OPM